api/tests/routes/backoffice/offers_test.py

```diff
@@ -55,6 +56,7 @@ def offers_fixture(criteria) -> tuple:
         venue__postalCode="47000",
         venue__departementCode="47",
         subcategoryId=subcategories.MATERIEL_ART_CREATIF.id,
+        author=factory.SubFactory(users_factories.ProFactory),
```

Comment:
Je suis surpris que tu utilises `SubFactory` ici, en général on appelle directement une factory dans les tests
```suggestion
        author=ProFactory(),
```

Reply:
Si je le fais ainsi, j'ai une erreur de type : 
`AttributeError: type object 'ProFactory' has no attribute '_sa_instance_state'`

Reply:
Je viens de tester sur ta branche, en remplaçant dans ce fichier tous les  : 
`author=factory.SubFactory(users_factories.ProFactory),`
par : 
`author=users_factories.ProFactory(),`
je n'ai aucune erreur !

Reply:
J'ai retenté et en effet je n'ai plus l'erreur, impossible de comprendre ce que j'ai fais  mais en tout cas c'est modifié :)

Reply:
sais tu par hasard ce qui créer ce genre d'erreur? Je me tape la meme erreur avec les collectivesoffers dont je m'occupe au niveau du test et le seul element de mon code faisant partie de la stack est : 
```
tests/routes/backoffice/collective_offers_test.py:42: in collective_offers_fixture
    collective_offer_1 = educational_factories.CollectiveStockFactory(
```
tout le reste justement fait partie des packages sqlalchemy

Reply:
j'ai compris, j'ai betement oublié la parenthese !